.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - run: npm ci
       - run: npm run build
       - run: npm run lint
-      - run: npm run test
+      - run: npx cross-env NODE_OPTIONS="--max-old-space-size=4096" npm run test
```

Comment:
I wouldn't do this. The problem is most likely that you upgraded something in the package-lock (did you perhaps run `npm audit fix`? 

There's a package in there somewhere that isn't happy with the current node version. 

Can you roll back the package-lock, delete node_modules, and run `npm install` again?

Reply:
I tried to "manually" revert back to the original package.json and commit the newly generated package-lock.json. This caused some other errors where the dependencies were missing some exported functions.

I ended up just running `git checkout master -- package.json package-lock.json` and committed that. That should roll back all the changes on this branch to master for any package*.json changes.

The first run of checks failed because 1 unit tests did not pass on the ubuntu CI. I reran it and all the tests pass.

Have you had trouble with the test show in the attached screenshot?
<img width="826" alt="Screenshot 2024-11-23 at 3 13 30 AM" src="https://github.com/user-attachments/assets/fe30d625-e8ae-46ff-aa35-232c9f3d0e94">